core/controllers/practice_sessions_test.py

```diff
@@ -49,14 +49,19 @@ def setUp(self):
             self.topic_id, 'public_topic_name', 'abbrev')
         self.topic.uncategorized_skill_ids.append(self.skill_id1)
         self.topic.subtopics.append(topic_domain.Subtopic(
-            1, 'subtopic_name', [self.skill_id2]))
+            1, 'subtopic_name', [self.skill_id2], 'image.svg',
+            constants.ALLOWED_THUMBNAIL_BG_COLORS['subtopic'][0]))
         self.topic.next_subtopic_id = 2
-        self.topic.thumbnail_filename = 'Topic.png'
+        self.topic.thumbnail_filename = 'Topic.svg'
+        self.topic.thumbnail_bg_color = (
+            constants.ALLOWED_THUMBNAIL_BG_COLORS['topic'][0])
         topic_services.save_new_topic(self.admin_id, self.topic)
 
         self.topic = topic_domain.Topic.create_default_topic(
             self.topic_id_1, 'private_topic_name', 'abbrev')
-        self.topic.thumbnail_filename = 'Topic.png'
+        self.topic.thumbnail_filename = 'Topic.svg'
```

Comment:
Maybe add a test for a different file format if we are supporting thumbnails in formats other than svg